src/ensembl/src/content/app/help/Help.scss

```diff
@@ -23,10 +23,24 @@
 
 .main {
   height: 100%;
-  padding-left: 80px;
-  padding-top: 44px;
+  display: grid;
+  padding-left: 44px;
+  padding-top: 25px;
+  grid-template-areas: "breadcrumbs breadcrumbs navigation"
+                       "space  article  article";
```

Comment:
If you want to skip a column, the css-grid convention is to use a dot ([docs](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas#leaving_a_grid_cell_empty))

Reply:
That's done